index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
```

Comment:
:+1:  - wady, zarzuty, cokolwiek informującego, że to nie jest opis całego fw, a tylko odpowiedź na pytanie "co wy się tak tego lrv czepiacie"